platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/builder/PlatformComponentBuilder.java

```diff
@@ -587,11 +588,23 @@ public PlatformComponentBuilder withConsensusEventStream(@NonNull final Consensu
     @NonNull
     public ConsensusEventStream buildConsensusEventStream() {
         if (consensusEventStream == null) {
+
+            // Required for conformity with legacy behavior. This sort of funky logic is normally something
+            // we'd try to move away from, but since we will be removing the CES entirely, it's simpler
+            // to just wait until the entire component disappears.
+            final Address address = blocks.initialState().get().getAddressBook().getAddress(blocks.selfId());
+            final String consensusEventStreamName;
+            if (!address.getMemo().isEmpty()) {
+                consensusEventStreamName = address.getMemo();
+            } else {
+                consensusEventStreamName = String.valueOf(blocks.selfId());
+            }
+
             consensusEventStream = new DefaultConsensusEventStream(
                     blocks.platformContext(),
                     blocks.selfId(),
                     (byte[] data) -> new PlatformSigner(blocks.keysAndCerts()).sign(data),
-                    "" + blocks.selfId().id(),
+                    consensusEventStreamName,
```

Comment:
It looks to me that it would be quite easy to add a unit tests to the given change.

Reply:
This already merged to develop, this is just a cherry pick. Was developed without tests because this issue is currently causing devops grief -- intent was to get a fix out the door ASAP. I created a ticket to add this test as a follow up. https://github.com/hashgraph/hedera-services/issues/13232